cmd/endpoint.go

```diff
@@ -349,14 +349,27 @@ func CreateEndpoints(serverAddr string, args ...string) (string, EndpointList, S
 					err = fmt.Errorf("path '%s' can not be served by different port on same address", endpoint.Path)
 					return serverAddr, endpoints, setupType, err
 				}
-
 				pathIPMap[endpoint.Path] = IPSet.Union(hostIPSet)
 			} else {
 				pathIPMap[endpoint.Path] = hostIPSet
 			}
 		}
 	}
 
+	// Check whether same path is used for more than 1 local endpoints.
+	{
+		localPathMap := make(map[string]int)
```

Comment:
Use `set.StringSet()` which makes things clean
```go
localPathSet = set.CreateStringSet()
for _, endpoint := range endpoints {
    if !endpoint.IsLocal {
        continue
    }
    if  localPathSet.Contains(endpoint.Path) {
        err = fmt.Errorf("path '%s' cannot be served by different address on same server", endpoint.Path)
        return ...
    }
    localPathSet.Add(endpoint.Path)
}

Reply:
Yes but this doesn't address > 2 @balamurugana we cannot use set.StringSet()